server/utils/validation/loginValidation.js

```diff
@@ -0,0 +1,14 @@
+const yup = require('yup');
+
+const schema = yup.object().shape({
+  email: yup.string().email().required(),
+  password: yup.string().min(4).required(),
+});
+
+const loginValidation = (email, password) =>
```

Comment:
what is the point of wrapping a function inside another function?
I think you should export the schema then you can call `isValid` function directly